docker/registry/internal/acr.go

```diff
@@ -29,6 +29,10 @@ func normalizeRepoDetailsAzure(repoDetails *docker.ImageRepoDetails) {
 	}
 }
 
+func (c *azureContainerRegistry) String() string {
```

Comment:
I might be confused but the stringer interface just returns the domain and no other information?

Reply:
Yep, its just to identify the repository ATM for tests.